pytorch_lightning/trainer/connectors/data_connector.py

```diff
@@ -220,7 +220,6 @@ def _worker_check(self, dataloader: DataLoader, name: str) -> None:
 
         # ddp_spawn + num_workers > 0 don't mix! tell the user
         if dataloader.num_workers > 0 and using_spawn:
-            # checks for the attr persistent_workers available in pytorch >= 1.7
             if hasattr(dataloader, "persistent_workers"):
```

Comment:
Do we still need to check if dataloader has persistent_workers? (I think the check should've been removed when dropping PyTorch 1.6 support.)
```suggestion
```

https://pytorch.org/docs/1.6.0/data.html#torch.utils.data.DataLoader
https://pytorch.org/docs/1.7.0/data.html#torch.utils.data.DataLoader
https://pytorch.org/docs/master/data.html#torch.utils.data.DataLoader